.github/workflows/node.js.yml

```diff
@@ -0,0 +1,51 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```

Comment:
Why only 14.x version? I think 15.x would be worth checking too as it is current release.

Reply:
Yes, sure.

But I think that if I do not have tests yet in my code, it is not important.

And Node.js 15 is odd-number version, so not for production using.
Now Node.js 14  STL is enough for now.